### PR TITLE
Localization support

### PR DESCRIPTION
**Pull request by kristofbolyai**
For some reason translations are not working, not yet sure why.

### CODE REVIEW

**Comment by kristofbolyai** [comment]
Issue: Translations does not load in Fabric dev environment, works in production. Forge works as expected.

**Comment by kristofbolyai** [comment]
This PR is ready to merge imo. Let me know if I missed any translations. The Fabric dev env issue could be fixed, but if noone has an idea, let's just make a separate issue regarding that.

File: common/src/main/resources/assets/wynntils/lang/en_us.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "feature.wynntils.itemGuess.name": "Item Guess Feature",
+  "feature.wynntils.itemGuess.possibilities": "§a- §7Possibilities: %s",
+  "feature.wynntils.itemStatInfo.name": "Item Statistics Info Feature",
+  "feature.wynntils.playerGhostTransparency.name": "Transparent Player Ghost Feature",
+  "feature.wynntils.soulPointTimer.name": "Soul Point Timer Feature",
+  "feature.wynntils.wynncraftButton.name": "Wynncraft Button Feature",
+  "featureDebug.wynntils.connectionProgress.name": "Connection Progress Feature",
+  "featureDebug.wynntils.keyBindTest.name": "Connection Progress Feature",
+  "featureDebug.wynntils.packetDebugger.name": "Packet Debugger Feature"
+}
```
Comment (magicus) [inline common/src/main/resources/assets/wynntils/lang/en_us.json]: Nit: Last line is missing newline

**Review by magicus — DISMISSED** [review]
Fix the nit, then it looks good. I agree that we can fix the fabric issue separately.

**Review by magicus — APPROVED** [review]

**Pull request merged**